src/quacc/calculators/espresso/espresso.py

```diff
@@ -381,29 +380,29 @@ def __init__(
         self.preset = preset
         self.parallel_info = parallel_info
         self.kwargs = kwargs
-        self._user_calc_params = {}
+        self.user_calc_params = {}
```

Comment:
Any reason for making this user-visible? It's not supplied by the user and is only used internally.

Reply:
It is accessed in _base.py, DeepSource was complaining that a "_" was accessed, the only reason I changed it

Reply:
Thanks, I'll add an override in deepsource.